superset/assets/visualizations/chord.jsx

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable no-param-reassign */
+import d3 from 'd3';
+import { category21 } from '../javascripts/modules/colors';
+import './chord.css';
+
+function chordViz(slice, json) {
+  slice.container.html('');
+
+  const div = d3.select(slice.selector);
+  const nodes = json.data.nodes;
```

Comment:
`json.data` corresponds exactly to what is returned in the backend `ChordViz.get_data` lower in this PR.